src/test/obj_ulog_prealloc/Makefile

```diff
@@ -0,0 +1,41 @@
+#
```

Comment:
I'd prefer if any new tests, especially this simple, were written in Python.

Reply:
Good point. We will change it.
\+ @rborawski